File: 210807 폴더 정리/practice3/app/src/main/java/com/example/practice3/MainActivity.java
package com.example.practice3;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.ListView;

import java.util.ArrayList;

public class MainActivity extends AppCompatActivity {

    ListView exampleList;
    ArrayList<String> dataSample;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        dataSample = new ArrayList<String>();

        dataSample.add("Test1");
        dataSample.add("Test2");
        dataSample.add("Test3");
        dataSample.add("Test1");
        dataSample.add("Test2");
        dataSample.add("Test3");
        dataSample.add("Test1");
        dataSample.add("Test2");
        dataSample.add("Test3");
        dataSample.add("Test1");
        dataSample.add("Test2");
        dataSample.add("Test3");
        dataSample.add("Test1");
        dataSample.add("Test2");
        dataSample.add("Test3");
        dataSample.add("Test1");
        dataSample.add("Test2");
        dataSample.add("Test3");
        dataSample.add("Test1");
        dataSample.add("Test2");
        dataSample.add("Test3");

        exampleList = findViewById(R.id.listView);
        ButtonListAdapter buttonListAdapter = new ButtonListAdapter(this, dataSample);

        exampleList.setAdapter(buttonListAdapter);


    }


}

File: 210807 폴더 정리/listviewcheckbox/app/src/main/java/com/example/listviewcheckbox/MainActivity.kt
package com.example.switchbutton2

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.recyclerview.widget.LinearLayoutManager
import com.example.listviewcheckbox.databinding.ActivityMainBinding


class MainActivity : AppCompatActivity() {

    private lateinit var binding: ActivityMainBinding
    private var userRecyclerViewAdapter: UserRecyclerViewAdapter? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)

        setUpRecyclerView()
    }

    private fun setUpRecyclerView(){
        binding.userRecyclerView.apply {
            userRecyclerViewAdapter = UserRecyclerViewAdapter()
            adapter = userRecyclerViewAdapter
            var layoutManager = LinearLayoutManager(this@MainActivity)
        }

    }

}